### ISSUE TITLE
`buildConfig()` arguments

### ISSUE THREAD

**Issue by mackerman44** [KevinSee/PITcleanr#31]
In the commit b5dbe8d, I changed the default for the `array_suffix =` argument to "UD", but left "A0B0" as an option. Once we get this branch to operate using "UD" as the default, consider removing "A0B0" as an option.

Related, the `array_suffix =` argument could eventually be changed to something like `max_arrays =` or ??? where `max_arrays = 2` would assign suffixes of _U and _D and `max_arrays = 3` would assign suffixes of _U, _M, and _D.

In the case that e.g., `max_arrays = 2` it would be nice to have the option to assign middle arrays to either _U or _D, perhaps depending on another argument. Something to consider.

**Comment by KevinSee** [comment]
I agree, once "UD" is up and running and confirmed to work, let's delete the "A0B0" option. 

The _U, _M, _D, (or more, if `max_arrays = 4`) option is great, and I believe that should work with most functions in `PITcleanr`. Might need some additional work or checking with the `addDirection()` and `filterDetections()` functions, to see how a middle array (or multiple middle arrays) might impact those.

**Comment by KevinSee** [comment]
I'm going to mark this issue as an enhancement, instead of to-do, because the rest of the package currently works with the UD option.
